CWE-94-Python_darwin/askervin_python-il/il.py

```
@@ -0,0 +1,480 @@
+            dictionary: use the dictionary as the il library.
+            The default is the filename + ".il" of the module in
+            which this decorator is used.
+
+      compiler_opts (list of strings, optional):
+            options passed to assembly compiler
+
+    See help(il.asm) for call convention.
+    '''
+    _lib = _load_lib(lib)
+    key = hashlib.sha1(code.encode("utf-8")).hexdigest()
+    if not key in _lib:
```

Ruff (0.13.2)

Test for membership should be `not in`

Convert to `not in`

(E713)